main.tf

```diff
@@ -471,7 +471,7 @@ resource "helm_release" "castai_pod_pinner" {
   depends_on = [helm_release.castai_agent]
 
   lifecycle {
-    ignore_changes = [set, version]
+    ignore_changes = [version]
```

Comment:
removing `set` will cause a drift AFAIK 

TF helm provider install helm release with replica_set 0. each apply will try to make sure it is 0.

Reply:
We tested this manually, and during `tf plan` it did not detect drifts across 5 tests. It only detects drifts using `tf plan -refresh-only`, which does not show up for customer operations. So as far as I understand, under normal operations the users should not see this as a drift.